app/src/main/java/com/kdev/consultacep/overview/DadosViewModel.kt

```diff
@@ -52,6 +52,11 @@ class DadosViewModel : ViewModel() {
                 println("********** MENSAGEM DE ERRO !!!: ${t.message} ***********")
             }
         }
+
+    }
+
+    fun getCep() : String? {
```

Comment:
qual a necessidade desse metodo?

Reply:
Esse método foi a tentativa de trazer os dados antes de eu conhecer o observer. A minha idéia era fazer um getter para cada atributo. Esqueci de remover hahahaha agora vou tirar ele.

Reply:
tendi, mas não tem muita necessidade mesmo de tu ter gentes para cada atributo, seria apenas poluição da classe, mas caso tu queira enviar os dados "mais mastigados" poderia usar uma sealed class:

`sealed class Presentation {
    object Loading : Presentation()
    data class Error(val message: String) : Presentation()
    data class Success(val cep: String, val ...) : Presentation()
}`

com isso tu pode criar um liveData do tipo da sealed class e no Fragment pode resolver via when

`when (presentation.value) {
        Presentation.Loading -> {
        }
        is Presentation.Error -> {
        }
        is Presentation.Success -> {
        }
    }`